sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -77,6 +80,20 @@ class DatalabAppElement extends Polymer.Element {
     this.page = page || 'files';
   }
 
+  /**
+   * on changes to the page property, resolve the new page's uri, and
```

Comment:
Please use sentence case (initial caps in these two lines) and punctuation (period on next sentence) in full-sentence comments. Also, function comments should use verb phrases as would appear after the implied "This function...", so in this case, "resolves" rather than "resolve", "tells" rather than "tell".